tools/downloader/topologies_test.py

```diff
@@ -9,13 +9,7 @@
 from openvino.inference_engine import IECore
 
 class TestTopologies(unittest.TestCase):
-
-    download_path = Path(os.getenv('OPEN_MODEL_ZOO_MODELS_DIR', Path('.').resolve() / 'models'))
-    ssd = omz.load_model('mobilenet-ssd')
-
-    def test_load_model(self):
-        self.assertTrue(os.path.exists(
-            os.path.join(self.download_path, 'public', 'mobilenet-ssd', 'mobilenet-ssd.caffemodel')))
+    ssd = omz.load_model('mobilenet-ssd', 'FP32')
```

Comment:
Why we need to specify precision for non-IR model? We should do it while `get_ir` instead